backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -99,16 +99,27 @@ def destroy
 
     if destroy_result
       invoke_callbacks(:destroy, :after)
-      flash[:success] = flash_message_for(@object, :successfully_removed)
       respond_with(@object) do |format|
-        format.html { redirect_to location_after_destroy }
-        format.js   { render partial: "spree/admin/shared/destroy" }
+        message = flash_message_for(@object, :successfully_removed)
+        format.html do
+          flash[:success] = message
+          redirect_to location_after_destroy
+        end
+        format.js do
+          render plain: message
```

Comment:
How do we show the success message in success case for a javascript request? The `.delete-resource` click handler only sets the message in error case not success case. https://github.com/solidusio/solidus/blob/master/backend/app/assets/javascripts/spree/backend/admin.js#L89

Can we keep the partial or update the handler to it also supports flash messages?

Reply:
Fixed, thank you!